src/plugins/vis_type_vislib/public/vislib/components/tooltip/_pointseries_tooltip_formatter.js

```diff
@@ -53,7 +54,15 @@ export function pointSeriesTooltipFormatter() {
 
     if (datum.y !== null && datum.y !== undefined) {
       const value = datum.yScale ? datum.yScale * datum.y : datum.y;
-      addDetail(currentSeries.label, currentSeries.yAxisFormatter(value));
+      let label = currentSeries.label;
+
+      // For stacked charts the y axis data is only available in the raw table
+      const tableColumns = get(datum, 'yRaw.table.columns');
```

Comment:
I would prefer `datum?.yRaw?.table?.columns` to loading `lodash` just for this.

Reply:
Good callout. Still need wrap my head around the fact that we now have opt chaining in Javascript. I'll leave this in for now unless there is another iteration needed since this is mostly just a stylistic change.

Reply:
Agreed on this. Are we planning on changing this or no because there are no other changes?